plugin/core/open.py

```diff
@@ -28,10 +43,9 @@ def open_file(
     # to open as a separate view).
     view = window.find_open_file(file)
     if view:
-        opens_in_desired_group = not bool(flags & sublime.FORCE_GROUP) or \
-                                 window.get_view_index(view)[0] == window.active_group()
-        opens_in_side_by_side = bool(flags & (sublime.ADD_TO_SELECTION | sublime.REPLACE_MRU))
-        return_existing_view = opens_in_desired_group and not opens_in_side_by_side
+        return_existing_view = _return_existing_view(
+            flags, window.get_view_index(view)[0], window.active_group(), group
+        )
         if return_existing_view:
```

Comment:
Redundant variable. Can be just:

```suggestion
        if _return_existing_view(flags, window.get_view_index(view)[0], window.active_group(), group):
```

or

```suggestion
        if _return_existing_view(flags, window.get_view_index(view)[0] == window.active_group(), group):
```

if you follow my other suggestion.